object.go

```diff
@@ -292,36 +295,72 @@ func (d *Decoder) decClassDef() (interface{}, error) {
 	return classInfo{javaName: clsName, fieldNameList: fieldList}, nil
 }
 
-func findField(name string, typ reflect.Type) ([]int, error) {
+type fieldInfo struct {
+	indexes []int
+	field   reflect.StructField
+}
+
+// map[rType][fieldName]indexes
+var _findFieldCache sync.Map
+
+func findField(name string, typ reflect.Type) (indexes []int, field reflect.StructField, err error) {
+	typCache, _ := _findFieldCache.Load(typ)
+	if typCache == nil {
+		typCache = &sync.Map{}
```

Comment:
Why use `&Sync.Map{}`, not just `Sync.Map{}`?

Reply:
Sync.Map is a structure which may lead some unnecessary copy

Reply:
Sync.Map is a structure which may lead some unnecessary copy

Reply:
Ok.